registry/diagnosis/collectlogs.sh

```diff
@@ -37,13 +37,25 @@ done
 
 mkdir -p ${LOGDIRECTORY}/containers
 echo "[$(date +%Y%m%d%H%M%S)][INFO] Collecting registry container logs."
+
+i=0
 for cid in $(sudo docker ps -a -q --no-trunc)
 do
+    echo "[$(date +%Y%m%d%H%M%S)][INFO] Getting cname of ${cid}."
```

Comment:
`cname` is just a variable name, `name` would be better

Reply:
Sure. change it to container name.